<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBInformation</class>
 <widget class="QWidget" name="DBInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBInformation</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QLabel" name="driveLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>驱动程序</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>driveComboBox</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="hostNameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>主机名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>hostLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="dataBaseLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据库名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>databaseLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>userNameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>passwordLineEdit</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="driveComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>QMYSQL (MySQL驱动)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QOCI (Oracle调用接口驱动程序)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QODBC (ODBC驱动程序（包括Microsoft SQL Server）)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QDB2 (IBM DB2)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QIBASE (Borland InterBase驱动程序)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QPSQL (PostgreSQL驱动)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QSQLITE (SQLite版本3或更高版本)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QSQLITE2 (SQLite版本2)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QTDS (Sybase Adaptive Server)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="hostLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="databaseLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="tableLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>表名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>userNameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="tableLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>不能修改项</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
